modules/apps/batch-engine/batch-engine-test/src/testFunctional/tests/dependencies/objectDefinition/export.jsonl

```diff
@@ -0,0 +1,2 @@
,"businessType":"Date","defaultValue":"","externalReferenceCode":"36a8da56-dd7e-61d2-42ae-4a1b72a0c203","id":43684,"indexed":false,"indexedAsKeyword":false,"indexedLanguageId":"","label":{"en_US":"Create Date"},"listTypeDefinitionExternalReferenceCode":"","name":"createDate","objectFieldSettings":[],"required":false,"state":false,"system":true,"type":"Date"},{"DBType":"String","businessType":"Text","defaultValue":"","externalReferenceCode":"77727124-66dc-e6bf-5b4b-54948cbb8999","id":43682,"indexed":false,"indexedAsKeyword":false,"indexedLanguageId":"","label":{"en_US":"Author"},"listTypeDefinitionExternalReferenceCode":"","name":"creator","objectFieldSettings":[],"required":false,"state":false,"system":true,"type":"String"},{"DBType":"String","businessType":"Text","defaultValue":"","externalReferenceCode":"a41e1b38-a129-0ee0-3e89-c202fed2f390","id":43686,"indexed":false,"indexedAsKeyword":false,"indexedLanguageId":"","label":{"en_US":"External Reference Code"},"listTypeDefinitionExternalReferenceCode":"","name":"externalReferenceCode","objectFieldSettings":[],"required":false,"state":false,"system":true,"type":"String"},{"DBType":"Long","businessType":"LongInteger","defaultValue":"","externalReferenceCode":"b232a830-4678-332a-9120-342c5e50494a","id":43688,"indexed":true,"indexedAsKeyword":true,"indexedLanguageId":"","label":{"en_US":"ID"},"listTypeDefinitionExternalReferenceCode":"","name":"id","objectFieldSettings":[],"required":false,"state":false,"system":true,"type":"Long"},{"DBType":"Date","businessType":"Date","defaultValue":"","externalReferenceCode":"1679874d-f724-42a4-f241-0008ae12db3c","id":43690,"indexed":false,"indexedAsKeyword":false,"indexedLanguageId":"","label":{"en_US":"Modified Date"},"listTypeDefinitionExternalReferenceCode":"","name":"modifiedDate","objectFieldSettings":[],"required":false,"state":false,"system":true,"type":"Date"},{"DBType":"String","businessType":"Text","defaultValue":"","externalReferenceCode":"dfe16957-e219-fd42-c754-4f6188403e94","id":43708,"indexed":true,"indexedAsKeyword":false,"indexedLanguageId":"en_US","label":{"en_US":"name"},"listTypeDefinitionExternalReferenceCode":"","name":"name","objectFieldSettings":[{"name":"showCounter","value":"false"}],"required":false,"state":false,"system":false,"type":"String"},{"DBType":"String","businessType":"Text","defaultValue":"","externalReferenceCode":"0696b345-c2c2-e305-061c-e6f7d32ad3df","id":43692,"indexed":false,"indexedAsKeyword":false,"indexedLanguageId":"","label":{"en_US":"Status"},"listTypeDefinitionExternalReferenceCode":"","name":"status","objectFieldSettings":[],"required":false,"state":false,"system":true,"type":"String"}],"objectLayouts":[],"objectRelationships":[],"objectViews":[],"panelCategoryKey":"","parameterRequired":false,"pluralLabel":{"en_US":"TestObjects"},"portlet":false,"restContextPath":"/o/c/testobjects","scope":"company","status":{"code":0,"label":"approved","label_i18n":"Approved"},"system":false,"titleObjectFieldName":"createDate"}
```

Comment:
to me it looks okay, but maybe Brian will ask us to make it more readable 🤔 Not sure here 😅

Reply:
That's right, maybe I can sf manually first :smile: